extensions/cms/src/cmsResource/tree/treeProvider.ts

```diff
@@ -63,18 +63,22 @@ export class CmsResourceTreeProvider implements TreeDataProvider<TreeNode>, ICms
 			if (registeredCmsServers && registeredCmsServers.length > 0) {
 				this.isSystemInitialized = true;
 				// save the CMS Servers for future use
-				await this._appContext.apiWrapper.setConfiguration(registeredCmsServers);
+				let toSaveCmsServers = JSON.parse(JSON.stringify(registeredCmsServers));
+				toSaveCmsServers.forEach(server => {
+					server.ownerUri = undefined,
+						server.connection.options.password = '';
+				});
+				await this._appContext.apiWrapper.setConfiguration(toSaveCmsServers);
 				return registeredCmsServers.map((server) => {
 					return new CmsResourceTreeNode(
 						server.name,
 						server.description,
 						server.ownerUri,
 						server.connection,
 						this._appContext, this, null);
-				});
+				}).sort();
```

Comment:
>.sort(); [](start = 6, length = 8)

Does this sort do the right thing with regards to using the names to do the sorting?